drgn_tools/dentry.py

```diff
@@ -381,24 +381,44 @@ def __file_type(mode: Object) -> str:
     return "UNKN"
 
 
-def ls(prog: Program, directory: str, count: bool = False) -> None:
+def ls(
+    prog: Program,
+    directory: str,
+    parent_dentry: Object,
+    depth: int,
```

Comment:
This makes these two new arguments _required_, which makes the ls function much less convenient to call from the CLI. Let's make these optional:
```suggestion
    parent_dentry: Optional[Object] = None,
    depth: int = 0,
```